spice/deps/helper_cb.py

```diff
@@ -27,12 +27,12 @@
 import os
 import logging
 import argparse
+from PIL import Image, ImageDraw
```

Comment:
PEP8 standard. It is a good way to write python code.

https://www.python.org/dev/peps/pep-0008/#imports

Plus

https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports

Use imports for packages and modules only.

Something says me, that Image / ImageDraw are classes.

And

PIL is not standard python library.

from PIL import Image, ImageDraw

ImportError: No module named PIL